packages/ui/src/app/components/PageLayout.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { ReactNode } from 'react'
+import styled, { css } from 'styled-components'
+
+import { PageContent } from '../../common/components/page/PageContent'
+import { Breadcrumbs } from '../../common/components/page/Sidebar/Breadcrumbs/Breadcrumbs'
+import { breadcrumbsOptions } from '../constants/breadcrumbs'
+
+export interface PageLayoutProps {
+  lastBreadcrumb?: string
+  header?: ReactNode
+  main?: ReactNode
+  lowSidebar?: ReactNode
+  highSidebar?: ReactNode
+  footer?: ReactNode
+}
+
+export const PageLayout = ({
+  header,
+  main: content,
+  lowSidebar,
+  highSidebar,
+  footer,
+  lastBreadcrumb,
+}: PageLayoutProps) => (
+  <PageContent>
+    <Breadcrumbs lastBreadcrumb={lastBreadcrumb} breadcrumbsOptions={breadcrumbsOptions} />
+    <PageLayoutComponent
+      header={header}
+      main={content}
+      lowSidebar={lowSidebar}
+      highSidebar={highSidebar}
+      footer={footer}
+    >
+      {header}
+      {content}
+      {lowSidebar}
+      {highSidebar}
+      {footer}
```

Comment:
Maybe a distinction Layouts components would be better instead of one that does everything.

`PageLayout` => `PageWithHighSidebarLayout`, `PageWithLowSidebarLayout`, etc